setup.py

```diff
@@ -54,8 +54,8 @@
 ONNX_BUILD_TESTS = bool(os.getenv('ONNX_BUILD_TESTS') == '1')
 ONNX_DISABLE_EXCEPTIONS = bool(os.getenv('ONNX_DISABLE_EXCEPTIONS') == '1')
 
-DEBUG = bool(os.getenv('DEBUG'))
-COVERAGE = bool(os.getenv('COVERAGE'))
+DEBUG = bool(os.getenv('DEBUG', '1') == '1')
+COVERAGE = bool(os.getenv('COVERAGE', '1') == '1')
```

Comment:
It's not related to the main issue in this PR, but I think it would be better if we can sync how to set config variable to prevent confusion. I thought DEBUG=0 should build a release build yet it did not..